admin-dev/themes/new-theme/public/theme.rtlfix

```diff
@@ -1,3 +1,74 @@
+         url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: Vazir;
+    src: url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Bold.eot');
+    src: url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Bold.eot?#iefix') format('embedded-opentype'),
+         url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Bold.woff2') format('woff2'),
+         url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Bold.woff') format('woff'),
+         url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+@font-face {
+    font-family: Vazir;
+    src: url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Light.eot');
+    src: url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Light.eot?#iefix') format('embedded-opentype'),
+         url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Light.woff2') format('woff2'),
+         url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Light.woff') format('woff'),
+         url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v30.1.0/dist/Vazir-Light.ttf') format('truetype');
+    font-weight: 300;
+    font-style: normal;
+}
+body.lang-fa, .lang-fa h1, .lang-fa h2, .lang-fa .modal-title, .lang-fa h3, .lang-fa h4, .lang-fa h5, .lang-fa h6, .lang-fa .h1, .lang-fa .h2, .lang-fa .h3, .lang-fa .h4, .lang-fa .h5, .lang-fa .h6, .lang-fa .tooltip, .lang-fa .popover, .lang-fa .module-search-result-title {
+  font-family: Vazir,Tahoma,Helvetica,Arial,sans-serif !important;
+}
+/* Fix arrow direction in categories list */
+ul.category-tree .more > .checkbox:before, ul.category-tree .more > .radio:before, .material-choice-tree-container ul.choice-tree .collapsed > .checkbox:before, .material-choice-tree-container ul.choice-tree .collapsed > .radio:before {
+  content: "\E5CB";
+}
+/* Fix arrow direction for tooltip */
+.popover .arrow {
+  transform: scaleX(1) !important;
+}
+/* Fix menu placement for TinyMCE */
+.mce-window, .mce-menu {
+  right: auto !important;
+  left: 0;
+}
+/* Fix page navigation arrows */
+.page-item.previous.next .page-link:after, .page-item.next.next .page-link:after {
+  content: "\E314";
+}
+.page-item.previous.previous .page-link:after, .page-item.next.previous .page-link:after {
+  content: "\E315";
+}
+/* Fix attributes list in product page */
+#attributes-list .attribute-group .attributes .attribute .js-attribute-checkbox + .attribute-label .pretty-checkbox {
+  float: right !important;
+  margin-left: 0.4375rem;
+  margin-right: 0 !important;
+}
+/* Fix categories list in product page */
```

Comment:
Some of changes that I've made like this, just effects on a specific page (eg. product page), but theme_rtl.css loads globally in all pages.

I don't know if there is a way to make these changes in SCSS files or not (Is RTL variable exists in SASS?). It would be more professional and clean. 

By the way this changes are a fast solution for now.

Reply:
`float-right` is all about css, it's an helper class, using this or adding it to an SCSS files looks really the same to me tbh

Anyway, I don't know if we have an RTL variable inside the BO @PrestaShop/prestashop-maintainers, and also, @mparvazi can you check if the body as an RTL data-attribute, that could be the thing you want in case you want to add some CSS only on RTL

The benefit about having an rtl stylesheet is that we don't load unused styles everywhere in the BO on non-RTL languages, but it's very small anyway.

Reply:
@NeOMakinG Right! It can be fixed easily:

```
body.lang-rtl .float-right {
    float: left !important;
}

```
![Screenshot 2021-10-04 191126](https://user-images.githubusercontent.com/85633460/135881619-61b71045-bb91-4816-be98-97a1ac52c326.png)